mypy/test/teststubgen.py

```diff
@@ -104,11 +104,20 @@ def cases(self) -> List[DataDrivenTestCase]:
         return c
 
 
+def parse_args(line: str) -> Options:
+    if line.startswith('# opts:'):
```

Comment:
In other mypy tests this is called ``# flags:``, so that it would be better to be consistent.